test/nodetool/test_repair.py

```diff
@@ -256,6 +256,53 @@ def test_repair_failed(nodetool):
          "Repair session 1 failed"])
 
 
+def test_repair_all_three_keyspaces_failed(nodetool):
```

Comment:
At the risk of repeating myself, I would prefer that you explain what the test does in a comment in front of it (or a docstring, if one of the fans of that thing), not only in the commit message - whose text you can reuse.
Even if the test's name was self explanatory (and it really isn't!), you usually need more than a few words to understand the purpose of the test.

Reply:
Done in v2.